### ISSUE TITLE
508 Office Request: Update alt text of VA logo in header & footer

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12697]
## Description
We did make an update to this alt text in October, however the 508 Office is asking us to do this again, with more specific language.  I did confirm this request with them and this new request is a result of the office making a change in their own standards based on the OIG audit finding that was allowing people to violate the _WCAG 3.2.4 Consistent Identification_ criterion which states that components that are the same within the website should be identified the same.

The 508 office is now asking that we update the alt text for the VA logo in the header and footer to be "VA logo and Seal, U.S. Department of Veterans Affairs"

## Additional Context
Previous text if helpful: #11271
* https://github.com/department-of-veterans-affairs/content-build/pull/1336
* https://github.com/department-of-veterans-affairs/vets-website/pull/22486

## Acceptance Criteria
- [x] Update alt text of logo in header to "VA logo and Seal, U.S. Department of Veterans Affairs"
- [x] Update alt text of logo in footer to "VA logo and Seal, U.S. Department of Veterans Affairs"
- [x] Update alt text of logo in new homepage modal to "VA logo and Seal, U.S. Department of Veterans Affairs"
- [x] Accessibility lead review

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by jilladams** [comment]
This was a standards change at the 508 office, so not technically a defect, hence: not applying the defect label unless/until we hear otherwise that it needs to be treated as a defect.

**Comment by laflannery** [comment]
We do not control the Medallia survey so that would not be covered here, but we should be able to update the new homepage modal.

@jilladams and @wesrowe Can you confirm if updating the Logo on the modal with the updated alt text would require a separate ticket or just a new AC in this one?

**Comment by jilladams** [comment]
Just a new AC here - updating.

**Comment by artsymartha68** [comment]
Thanks! Any advice on how to create the Medallia modal issue ticket? Happy to help.

**Comment by artsymartha68** [comment]
I created the following ticket for the Medallia modal: https://github.com/department-of-veterans-affairs/va.gov-team/issues/54247

**Comment by jilladams** [comment]
@apisandipas this is a ticket where you can probably hop in and help once you're online. 
I tried to make these changes locally and push, and run into weird linting errors in vets-website. I hacked around that by putting up the changes directly via the Github UI, but the v-w PR prob won't be cleared to merge that way. Here are the errors: 

`vets-website`: 
```
✖ eslint --fix -c ./.eslintrc.changed.js:

/Users/jilladams/code/vets-website/src/applications/proxy-rewrite/partials/header.js
  1:1  error  Definition for rule '@department-of-veterans-affairs/remove-expanding-group' was not found  @department-of-veterans-affairs/remove-expanding-group

/Users/jilladams/code/vets-website/src/platform/site-wide/va-footer/components/Footer.jsx
  1:1   error  Definition for rule '@department-of-veterans-affairs/remove-expanding-group' was not found  @department-of-veterans-affairs/remove-expanding-group
  3:23  error  Unable to resolve path to module 'prop-types'                                               import/no-unresolved
  5:26  error  Unable to resolve path to module 'lodash'  
```

**Comment by jilladams** [comment]
Nevermind, that was a local issue. PRs passed tests fine.

**Comment by jilladams** [comment]
Verified in prod on both header, footer, and on an injected header site, at various breakpoints.

**Comment by laflannery** [comment]
@jilladams I confirmed the first 2 ACs but the 3rd (the alt text on the Modal logo) I do not see updated:
![image](https://user-images.githubusercontent.com/106678594/227974122-a654fd3e-9b5f-4409-8e16-b9b548b4d3bf.png)

**Comment by jilladams** [comment]
@laflannery good catch, apologies -- had that one mixed up with Medallia in my mind. Reopening to fix.

**Comment by laflannery** [comment]
Confirmed modal logo is now updated on prod - closing this one again
